confmodel/tests/test_fields.py

```diff
@@ -145,7 +145,7 @@ def assert_url(value,
         assert_url(self.field_value(field, 'foo'), path='foo')
         assert_url(self.field_value(field, u'foo'), path='foo')
         assert_url(self.field_value(field, u'foo\u1234'),
-                   path='foo\xe1\x88\xb4')
+                   path=u'foo\u1234')
```

Comment:
I'm thinking at this config level, you want to use unicode for everything - and deal with twisted's shortcomings elsewhere. If we encode as utf-8 here in Python 3, urlparse blows up. It either takes a unicode str or ascii encoded bytes.

How do you want to proceed with this ?